config/rbac/namespace_bindings/istio_namespace_role.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Why not define this as a `Role` only, all the resources is in `istio-namespace`, right?

Reply:
yeah good point since it is already configured manually not using the kubebuilder markers